### PR TITLE
feat: Support mapping C# exceptions to native formats 

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What and why?

Converting C# stacks to native formats allows us to map stack frames to C# files and lines using the IL2CPP mapping file. Without this, the file and line information is lost and only the method is usable.

We offer this mapping as a build option only when symbol files are output, because the native stack traces are basically useless without symbol files.

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests

### CODE REVIEW

File: samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle [hunk 1]
```diff
@@ -18,6 +18,7 @@ dependencies {
 
 // Android Resolver Exclusions Start
 android {
+	namespace "com.unity3d.player"
```
Comment (0xnm) [inline samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle]: should be Datadog-specific namespace

File: packages/Datadog.Unity/Runtime/iOS/DatadogiOSPlatform.cs [hunk 2]
```diff
@@ -116,6 +126,87 @@ public void ClearAllData()
...
+                    }
+
+                    if (imageLoadAddress <= 0)
+                    {
+                        // Couldn't find the image load address, so we can't supply the stack trace
+                        return null;
+                    }
+
+                    var moduleName = Path.GetFileNameWithoutExtension(imageName);
+                    // Strip off weird \u0001 character that appears at the end of module names
```
Comment (0xnm) [inline packages/Datadog.Unity/Runtime/iOS/DatadogiOSPlatform.cs:169]: does it appear always?

**Review by 0xnm — DISMISSED** [review]

File: samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle (hunk 1 above)
Comment (fuzzybinary) [reply samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle]: 🤔 Did this accidentally get reverted?

File: packages/Datadog.Unity/Runtime/iOS/DatadogiOSPlatform.cs (hunk 2 above)
Comment (fuzzybinary) [reply packages/Datadog.Unity/Runtime/iOS/DatadogiOSPlatform.cs:169]: It appears so yes. At least in all of my testing there's an extra character at the end of the module name.

**Review by 0xnm — APPROVED** [review]

**Pull request merged**